src/WidgetBase.ts

```diff
@@ -122,7 +122,6 @@ const boundAuto = auto.bind(null);
 /**
  * Main widget base for all widgets to extend
  */
-@diffProperty('registry', ignore)
```

Comment:
We don't need this special `diffProperty` for registry now it is not a supported property.